db/db_impl.cc

```diff
@@ -5544,6 +5544,29 @@ ColumnFamilyHandle* DBImpl::GetColumnFamilyHandle(uint32_t column_family_id) {
   return cf_memtables->GetColumnFamilyHandle();
 }
 
+void DBImpl::GetApproximateMemTableStats(ColumnFamilyHandle* column_family,
+                                         const Range* range,
+                                         int n, uint64_t* counts,
+                                         uint64_t* sizes) {
+  auto cfh = reinterpret_cast<ColumnFamilyHandleImpl*>(column_family);
+  auto cfd = cfh->cfd();
```

Comment:
We follow Google C++ Style, in which this auto is not recommended as it's not clear about the type in this context: https://google.github.io/styleguide/cppguide.html#auto